packages/iTwinUI-react/src/core/Select/Select.tsx

```diff
@@ -303,29 +395,69 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
             setIsOpen(false);
           }
         }}
+        setFocus={!multiple}
```

Comment:
shouldn't we manually focus first item if we are disabling this?

Reply:
We should, but as for now, dont know how

Reply:
Ok, I have moved weird hook from `Menu` which was reseting focused index. This fixes issue in multiple select.
I have tried to go through all other stories and havent noticed keyboard nav to be broken anywhere.